### PR TITLE
Update android 7.1.1 r25 to android 7.1.2 r36

### PR DESCRIPTION
**Pull request by lnjX**

### CODE REVIEW

**Comment by bhush9** [comment]
If you do this change, you would also need to merge the related tag in all the forks we have..

**Comment by lnjX** [comment]
Oh, didn't thought about that.

**Comment by lnjX** [comment]
This requires these PRs to be merged:
https://github.com/Halium/android_frameworks_av/pull/1
https://github.com/Halium/android_bionic/pull/2
https://github.com/Halium/android_external_skia/pull/1
https://github.com/Halium/android_frameworks_base/pull/1
https://github.com/Halium/android_frameworks_native/pull/1
https://github.com/Halium/android_hardware_libhardware/pull/2
https://github.com/Halium/android_hardware_qcom_fm/pull/1
https://github.com/Halium/android_system_core/pull/2
https://github.com/Halium/android_system_security/pull/1
https://github.com/Halium/android_system_netd/pull/1
https://github.com/Halium/android_vendor_cm/pull/1

Manifest to test it without the merges:
https://gist.github.com/LNJ2/ea7e1985ed542438cfc82a0c56051c86

File: default.xml [hunk 1]
```diff
@@ -67,7 +67,7 @@
   <project path="external/icu" name="android_external_icu" groups="pdk" remote="los" />
   <project path="external/iptables" name="android_external_iptables" groups="pdk" remote="los" /> 
   <project path="external/jemalloc" name="android_external_jemalloc" remote="los" groups="pdk,flo" />
-  <project path="external/jhead" name="platform/external/jhead" groups="pdk" remote="aosp" /> -->
+  <project path="external/jhead" name="platform/external/jhead" groups="pdk" remote="aosp" revision="refs/tags/android-7.1.2_r33" /> -->
```
Comment (lnjX) [inline default.xml:70]: This repository somehow has no r36 tag

File: default.xml [hunk 2]
```diff
@@ -214,7 +214,7 @@
   <project path="prebuilts/ndk" name="platform/prebuilts/ndk" groups="pdk" clone-depth="1" remote="aosp" />
   <project path="prebuilts/ninja/linux-x86" name="platform/prebuilts/ninja/linux-x86" groups="linux,pdk,tradefed" clone-depth="1" remote="aosp" />
   <project path="prebuilts/python/linux-x86/2.7.5" name="platform/prebuilts/python/linux-x86/2.7.5" groups="linux,pdk,pdk-cw-fs,pdk-fs" clone-depth="1" remote="aosp" />
-  <project path="prebuilts/tools" name="platform/prebuilts/tools" groups="pdk,tools" />
+  <project path="prebuilts/tools" name="platform/prebuilts/tools" groups="pdk,tools" revision="refs/tags/android-7.1.2_r33" />
```
Comment (lnjX) [inline default.xml:217]: The checkout of this repo with the r36 tag always failed, saying the remote hasn't sent all objects.

**Comment by bhush9** [comment]
@LNJ2 can you merge this?

**Comment by lnjX** [comment]
@bhush9 If you're not disagreeing, sure!

**Pull request merged**